lib/components/narrative/narrative-itineraries.js

```diff
@@ -448,7 +444,6 @@ class NarrativeItineraries extends Component {
         <NarrativeItinerariesHeader
           customBatchUiBackground={customBatchUiBackground}
           enabledSortModes={enabledSortModes}
-          errors={errors}
```

Comment:
Also remove:
- prop `onToggleShowErrors`
- the corresponding component state `showErrors`, and propagate as needed.
- prop `showingErrors` (remove prop from `NarrativeItinerariesHeader` too)

Reply:
good catch.